packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -343,6 +344,35 @@ export default class ImageEdit implements EditorPlugin {
         }
     }
 
+    /**
+     * Flip the image.
+     * @param image The image to be flipped
+     */
+    public flipImage(image: HTMLImageElement) {
+        this.image = image;
+        this.editInfo = getEditInfoFromImage(image);
+        this.editInfo.flippedImage = !this.editInfo.flippedImage;
+        this.createWrapper(ImageEditOperation.Rotate);
+        this.updateWrapper();
+        this.setEditingImage(null);
```

Comment:
I'm curious why we need to create wrapper than set editing image to null which will dispose the wrapper? Can we just not create wrapper at all?

Reply:
createWarpper creates the clonedImage that we use in the updateWrapper, we use updateWrapper to proper calculate the image size if rotated

Reply:
I see.